portfolio.html

```diff
@@ -27,5 +27,113 @@ <h2>I’am a Software Developer</h2>
       <li><a href="" class="socialMedia"><img src="images/Icon-Medium.png" alt="Medium Icon" class="Icons Icon5"></a></li>
     </ul>
   </section>
+  <section class="secondPage">
+    <h1 class="recentWorks">My Recent Works</h1>
+    <img src="images/yogaWoman.png" class="yogaImage" alt="Woman doing Yoga">
+    <h2 class="multiStories">Multi-Post Stories</h2>
+    <p class="secondPara">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <ul class="languageList">
+      <li class="languageList css">css</li>
+      <li class="languageList html">html</li>
+      <li class="languageList bootstrap">bootstrap</li>
+      <li class="languageList ruby">Ruby</li>
+    </ul>
+    <button class="seeButton">See Project</button>
+  </section>
+  <section class="projects">
+    <div class="cards">
+      <h3>Profesional Art Printing Data</h3> 
```

Comment:
The `See Project` button is wider for this card than the previous card. It's a good idea to follow the given design and also make sure there's uniformity in the flow of items on the page.
![Screenshot from 2021-09-17 16-42-19](https://user-images.githubusercontent.com/80149314/133802657-e7dce2a1-51e4-4917-bad5-0786d419fbae.png)